keyboards/ergo42/keymaps/koba/keymap.c

```diff
@@ -0,0 +1,130 @@
+/* Copyright 2018 Daisuke Kobayashi
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+enum {
+  _WIN,
+  _MACOS,
+  _LOWER,
+  _RAISE,
+  _FUNC
+};
+
+#define _______ KC_TRNS
```

Comment:
Lines 28-29 aren't necessary, as QMK defines these automatically for every keymap.

https://github.com/qmk/qmk_firmware/blob/35e76539e7257a1932ec76882892b999e21c9d4f/quantum/quantum_keycodes.h#L29-L31